.vscode/tasks.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build the appolo project",
+            "type": "shell",
+            // you can change the "build" option to others acording to the "apollo.sh" file
+            "command": "bash dev_build.sh build",
```

Comment:
maybe you can use the existing command here: `bash apollo_docker.sh build`

Reply:
Yes, your are right. I will change my code and submit again. Thank you for your valuable suggestions.